_targets.R

```diff
@@ -14,24 +14,24 @@ yaml_file <- "test_config.yml"
 
 # RUNNING {TARGETS}:
 #
-# run tar_make() to run the pipeline
-# run tar_visnetwork() to see visual summary
+# use the file 'run_targets.Rmd', which includes EE authentication.
 
 
 # Set up python virtual environment ---------------------------------------
 
 if (!dir.exists("env")) {
-  source("data_acquisition/src/pySetup.R")
+  tar_source("data_acquisition/src/pySetup.R")
```

Comment:
this was a MB suggestion - it doesn't actually change anything, but it's more 'targets'-y